src/yukon/backend/src/devserv/description/template/sess_description.json

```diff
@@ -0,0 +1,61 @@
+{
+        "log_file": "",
+        "interactions": [{
+                "bus": {
+                        "id": ""
+                },
+                "nodes": {
+                        "0": {
+                                "subject": {
+                                        "publish": {
+                                                "timestamp_start": "",
+                                                "timestamp_end": "",
+                                                "port_id": "",
+                                                "timeout": ""
+                                        },
+                                        "subscribe": {
+                                                "timestamp_start": "",
+                                                "timestamp_end": "",
+                                                "port_id": "",
+                                                "timeout": ""
+                                        }
+                                },
```

Comment:
I think the `subject`/`service` segregation is not needed. Publishing, subscribing, calling, and serving seems unambiguous enough.